dynawo/sources/Solvers/SolverIDA/DYNSolverIDA.cpp

```diff
@@ -134,8 +133,8 @@ SolverIDA::SolverIDA() {
   LS_ = NULL;
   // KINSOL solver Init
   //-----------------------
-  solverKINNormal_.reset(new SolverKIN());
-  solverKINYPrim_.reset(new SolverKIN());
+  solverKINNormal_.reset(new SolverKINAlgRestoration());
+  solverKINYPrim_.reset(new SolverKINAlgRestoration());
 
   lastRowVals_ = NULL;
   order_ = 0;
```

Comment:
The new attributes should be assigned a value in the default constructor

Reply:
Done